src/Silk.Data/Entities/Guild/User/UserEntity.cs

```diff
@@ -16,13 +16,15 @@ public class UserEntity
     /// <summary>
     /// The ID of the user's guild.
     /// </summary>
-    [Column("guild_id")]
+    [NotMapped]
     public Snowflake GuildID { get; set; }
 
-    /// <summary>
-    /// The guild this user belongs to.
-    /// </summary>
-    public GuildEntity Guild { get; set; } = null!;
+    // /// <summary>
+    // /// The guild this user belongs to.
+    // /// </summary>
+    // public GuildEntity Guild { get; set; } = null!;
+
+    public ICollection<GuildEntity> Guilds { get; set; }
```

Comment:
Could make type `List<GuildEntity>` but completely optional, fine as is

Reply:
Primarily used for EF Core; scared to change it. DTOs are definitely justified here.